deepset_cloud_sdk/api/config.py

```diff
@@ -7,13 +7,22 @@
 
 load_dotenv()
 
+# connection to deepset Cloud
 API_URL = os.getenv("API_URL", "https://api.cloud.deepset.ai/")
 API_KEY = os.getenv("API_KEY", "")
 
+# configuration to use a selectd workspace
+DEFAULT_WORKSPACE_NAME = os.getenv("DEFAULT_WORKSPACE_NAME", "default")
```

Comment:
personally I don't think we should provide a default workspace name - I think it increases the liklihood of things accidentally ending up in `default` workspaces without really giving any real benefit.

Reply:
Should we just raise an exception if no workspace is defined via env vars and not passed via arguments ? Would that be better ? 🤔

Reply:
I think passing the workspace as an argument makes more sense instead of an environment variable? Maybe it's a good question to ask a Sol, we can leave it as is for now and ask for some feedback